cmd/booster-http/run.go

```diff
@@ -152,6 +153,13 @@ var runCmd = &cli.Command{
 			return fmt.Errorf("parsing storage API endpoint: %w", err)
 		}
 
+		// Connect to nitro payments API
+		nitroApiInfo := cctx.String("api-nitro")
+		if err != nil {
+			return fmt.Errorf("parsing storage API endpoint: %w", err)
+		}
+		nitroClient, err := rpc.NewHttpRpcClient(nitroApiInfo)
+
```

Comment:
We would probably want to open a ledger channel here. Thankfully, no money needs to be sent on chain. 

Longer term, it could be done outside the constructor somehow.